marlowe-tutorial/doc/embedded-marlowe.adoc

```diff
@@ -0,0 +1,209 @@
+                    (If
+                       (ValueEQ
+                          (ChoiceValue
+                             (ChoiceId "choice" "alice")
+                             (Constant 42))
+                          (ChoiceValue
+                             (ChoiceId "choice" "bob")
+                             (Constant 42)))
+                       (If
+                          (ValueEQ
+                             (ChoiceValue
+                                (ChoiceId "choice" "alice")
+                                (Constant 42))
+                             (Constant 0))
+                          (Pay
+                             (AccountId 0 "alice")
+                             (Party "bob")
+                             (Constant 450) Refund) Refund)
+                       (When [
+                             (Case
+                                (Choice
+                                   (ChoiceId "choice" "carol") [
+                                   (Bound 1 1)]) Refund)
+                             ,
+                             (Case
+                                (Choice
+                                   (ChoiceId "choice" "carol") [
+                                   (Bound 0 0)])
+                                (Pay
+                                   (AccountId 0 "alice")
+                                   (Party "bob")
+                                   (Constant 450) Refund))] 100 Refund)))] 60
+                 (When [
+                       (Case
+                          (Choice
+                             (ChoiceId "choice" "carol") [
+                             (Bound 1 1)]) Refund)
+                       ,
+                       (Case
+                          (Choice
+                             (ChoiceId "choice" "carol") [
+                             (Bound 0 0)])
+                          (Pay
+                             (AccountId 0 "alice")
+                             (Party "bob")
+                             (Constant 450) Refund))] 100 Refund)))
+           ,
+           (Case
+              (Choice
+                 (ChoiceId "choice" "bob") [
+                 (Bound 0 1)])
+              ... similarly to the "alice" case above ...
+               
+----
+
+____
+*Exercises*
+
+What other abbreviations could you add to the contract at the top of the
+page?
+
+Can you spot any _functions_ that you could define to make the contract
+shorter, or more modular?
+____
+
+This example has shown how embedding in Haskell gives us a more
+expressive language, simply by reusing some of the basic features of
+Haskell, namely definitions of constants and functions. In the next
+tutorial you will learn about how to _exercise_ Marlowe contracts in ghci.
+
+==== Note
+
+* This contracts is contained in the modules
+https://github.com/input-output-hk/marlowe/blob/master/semantics-3.0/src/Language/Marlowe/Examples/EscrowSimpleV2.hs[`+EscrowSimpleV2.hs+`]
```

Comment:
I'd prefer to avoid absolute links to github - these get out of date very easily. Do we need this?

Reply:
This will be updated, the tutorial is a work in progress but is currently suitable for stable. I'll mention this to @simonjohnthompson